Makefile

```diff
@@ -88,6 +88,40 @@ test-initrd-only:
 list-distros:
 	@ $(ROOTFS_BUILDER) -l
 
+DESTDIR := /
+KATADIR := /usr/libexec/kata-containers
+OSBUILDER_DIR := $(KATADIR)/osbuilder
+INSTALL_DIR :=$(DESTDIR)/$(OSBUILDER_DIR)
+DIST_CONFIGS:= $(wildcard rootfs-builder/*/config.sh)
+
+SCRIPTS :=
+SCRIPTS += rootfs-builder/rootfs.sh
+SCRIPTS += image-builder/image_builder.sh
+SCRIPTS += initrd-builder/initrd_builder.sh
+
+FILES :=
+FILES += rootfs-builder/versions.txt
+FILES += scripts/lib.sh
+
+define INSTALL_FILE
+	echo "Installing $(abspath $2/$1)";
+	install -m 644 -D $1 $2/$1;
+endef
+
+define INSTALL_SCRIPT
+	echo "Installing $(abspath $2/$1)";
+	install -m 755 -D $1 $(abspath $2/$1);
+endef
+
+.PHONY: install
+install:
```

Comment:
I'd call this target something like `install-scripts`, since `install` is typically used to install a build artifact.